app/views/self_service/firms/_table_trading_names.html.erb

```diff
@@ -8,12 +8,7 @@
         <th class="firms-table__frn">
           <%= Firm.human_attribute_name(:fca_number) %>
         </th>
-        <th class="firms-table__principal">
-          <%= t('self_service.firms_index.principal_column_heading') %>
-        </th>
-        <th class="firms-table__principal">
-          <%= t('self_service.firms_index.principal_column_heading') %>
-        </th>
+        <th class="firms-table__action"></th>
```

Comment:
![image](https://cloud.githubusercontent.com/assets/1007202/8572554/21d040d2-2585-11e5-84a9-cdb74ad9342f.png)

It opens up a table heading cell, otherwise it'd just be a blank space. The class sets the width of the column.